README.md

```diff
@@ -27,11 +27,11 @@ To install the bindings via [Composer](https://getcomposer.org/), add the follow
   "repositories": [
     {
       "type": "vcs",
-      "url": "https://github.com/lilt/lilt-connector-sdk-php.git"
+      "url": "https://github.com/GIT_USER_ID/GIT_REPO_ID.git"
```

Comment:
i think a few things changed about the readme. if we provide these options to the generator i think it will fix these values

```sh
docker run --rm -v "${PWD}:/local" openapitools/openapi-generator-cli generate \
    --git-repo-id lilt-connector-sdk-php \
    --git-user-id lilt \
    -p packageName=lilt-connector-sdk \
    -p invokerPackage=LiltConnectorSDK \
    -i https://connectors-admin.lilt.com/docs/api/_specs/plugin_api.json \
    -g php \
    -o /local/
```